tests/test_operation.py

```diff
@@ -19,57 +20,61 @@ def test_operation(
     strategy.tend()
 
     # withdrawal
-    vault.withdraw({"from": accounts[0]})
-    assert pytest.approx(token.balanceOf(accounts[0]), rel=RELATIVE_APPROX) == amount
+    vault.withdraw({"from": user})
+    assert pytest.approx(token.balanceOf(user), rel=RELATIVE_APPROX) == user_balance_before
 
 
 def test_emergency_exit(
-    accounts, token, vault, strategy, strategist, amount, RELATIVE_APPROX
+    accounts, token, vault, strategy, user, strategist, amount, RELATIVE_APPROX
 ):
     # Deposit to the vault
-    token.approve(vault.address, amount, {"from": accounts[0]})
-    vault.deposit(amount, {"from": accounts[0]})
-    strategy.harvest()
+    token.approve(vault.address, amount, {"from": user})
+    vault.deposit(amount, {"from": user})
+    strategy.harvest({"from": strategist})
```

Comment:
The object strategy has been created using the strategist account, you don't need to specify it again.

Reply:
Reason I changed this one is to be more portable for testing against already-deployed strategies/vaults.